extensions/wikia/Discussions/controllers/EnableDiscussionsController.class.php

```diff
@@ -61,6 +61,19 @@ public function toggleVars() {
 		] ) );
 	}
 
+	public function toggleReadOnlyForum() {
```

Comment:
This controller will be available always in community wiki. That's why it is not in Forum related controller.